test/Chainweb/Test/Pact/RemotePactTest.hs

```diff
@@ -702,7 +705,7 @@ allocationTest iot nio = testCaseSteps "genesis allocation tests" $ \step -> do
       $ ObjectMap
       $ M.fromList
         [ (FieldKey "account", PLiteral $ LString "allocation02")
-        , (FieldKey "balance", PLiteral $ LDecimal 1_099_995.13) -- 1k + 1mm - gas
+        , (FieldKey "balance", PLiteral $ LDecimal 1_099_995.05) -- 1k + 1mm - gas
```

Comment:
I assume that release allocation got cheaper (in terms of gas) with pact 4.2. Is that correct?

Reply:
Seems more expensive in terms of gas, because there's less balance remaining.

Reply:
Yes, slightly more expensive.

Reply:
Why?